app/poros/search_results.rb

```diff
@@ -0,0 +1,27 @@
+class SearchResults
+  def initialize(user)
+    @user = user
+    @github_service = GithubService.new(@user.token, @user.github_user)
+  end
+
+  def repos
+    json = @github_service.repos
+    @repos = json.map do |repo|
+      Repo.new(repo[:name], repo[:html_url])
+    end
+  end
+
+  def followers
+    json = @github_service.followers
+    @followers = json.map do |user|
```

Comment:
super clean, this is helpful to see, gives me ideas of what to do with the youtube playlists